### ISSUE TITLE
Book Selection: Add book duration, % read

### ISSUE THREAD

**Issue by NeuroNex** [Ph1b/MaterialAudiobookPlayer#202]
The Album/Book selection screen is very nice. It would be really cool if to add some useful information in overlay on the book cover (the same way as the "playing" icon is placed on the current playing book). The useful info are:

1. The total duration of the book. Format hour minutes. Example: "16h 43 m" or "47h 28m"

2. The percentage of the book which is already read. This could be a thin color bar on the top edge of the book cover. 100%  = the entire width of the cover image. Let's assume that the user had already read 20% of a book, then the corresponding color "progression" bar is displayed with 20% of the cover width. Of course, this requires that the app must keep track of the read duration for each book.

This would be an excellent feature. So far, I only see it in [Listen Audiobook Player](https://play.google.com/store/apps/details?id=com.acmeandroid.listen)

**Comment by Ph1b** [comment]
I tried it out, but I prefere to have the UI cleaner.

**Comment by NeuroNex** [comment]
Hi,
When I first used the app [Listen Audiobook Player](https://play.google.com/store/apps/details?id=com.acmeandroid.listen). I thought that too (it's not a big deal to know the percentage read per book). But overtime, I realized that this is quite a useful clue. Actually I didn't think that I would jump from book to book like that quite often. In my opinion, if you implement that % read, the clutter on the GUI would not be that much, as the percentage read would take a thin color bar on top of each cover image, playing the role of a % read progression bar.

Here is a screenshot of the book selection screen from the "Listen Audiobook Player" app. Just for an example.

![listenaudiobook_library](https://cloud.githubusercontent.com/assets/10089894/7898571/0ec781ac-06d2-11e5-94fa-9cfce1edc1be.png)
